src/main/rx/ghst.c

```diff
@@ -291,16 +341,30 @@ STATIC_UNIT_TESTED float ghstReadRawRC(const rxRuntimeState_t *rxRuntimeState, u
 {
     UNUSED(rxRuntimeState);
 
-    // derived from original SBus scaling, with slight correction for offset (now symmetrical
-    // around OpenTx 0 value)
+    // Scaling 12bit channels (8bit channels in brackets)
+    //      OpenTx          RC   PWM (BF)
+    // min  -1024        0(  0)     988us
+    // ctr      0     2048(128)    1500us
+    // max   1024     4096(256)    2012us
+    //
+
+    // Scaling legacy (nearly 10bit)
+    // derived from original SBus scaling, with slight correction for offset
+    // now symmetrical around OpenTx 0 value
     // scaling is:
-    //      OpenTx   RC     PWM
-    // min  -1024    172    988us
-    // ctr  0        992    1500us
-    // max  1024     1811   2012us
+    //      OpenTx         RC   PWM (BF)
+    // min  -1024     172( 22)     988us
+    // ctr      0     992(124)    1500us
+    // max   1024    1811(226)    2012us
```

Comment:
what are those values in brackets?

Reply:
Those values are used for AUX channels on Ghost side. They are only 8 bit .